Documentation/ApiOverview/Backend/ContextualMenu.rst

```diff
@@ -31,15 +22,18 @@ Context menu rendering flow
 Markup
 ------
 
-The context menu is shown after click on the HTML element which has
+The context menu is shown after clicking on the HTML element which has
 :html:`class="t3js-contextmenutrigger"` together with :html:`data-table`,
 :html:`data-uid` and optional :html:`data-context` attributes.
 
 The JavaScript click event handler is implemented in the
-:js:`TYPO3/CMS/Backend/ContextMenu` requireJS module. It takes the
-data attributes mentioned above and executes an ajax call to the
+ES6 module :js:`@typo3/backend/context-menu.js`. It takes the
```

Comment:
Also here, can we link "ES6 module" to a page which explains it? For me, I would be lost now.

Reply:
From what I understand "ES6" is a common concept. There must be sources on this somewhere (TM) but I am just starting to learn about it myself. And the guys who made the changes didn't spend much time on documentation

Reply:
added links